apps/src/templates/GameButtons.jsx

```diff
@@ -3,6 +3,15 @@ var msg = require('../locale');
 var ProtectedStatefulDiv = require('./ProtectedStatefulDiv');
 var commonStyles = require('../commonStyles');
 
+var styles = {
+  instructionsInTopPane: {
+    // common.scss provides an :after selector that ends up adding 18px of height
+    // to gameButtons. We want to get rid of that when we have don't have
+    // instructions below game buttons
+    marginBottom: -18
```

Comment:
Not entirely clear to me why the :after selector takes up 18px, given that it specifies its height and line-height as 0px. As such, I don't love the approach I'm taking here, but it does seem to work.